lib/MetagenomeUtils/Utils/AMAUtils.py

```diff
@@ -0,0 +1,42 @@
+class AMAUtils():
+
+    def __init__(self, ws):
+        self.ws = ws
+
+    def _confirm_ws_type(self, ref):
+        if ref is None:
+            raise ValueError(" 'ref' argument must be specified.")
+        obj_info = self.ws.get_object_info3({
+            'objects': [{'ref': ref}]
+        })['infos'][0]
+        # check object type is 'KBaseMetagenome.AnnotatedMetagenomeAssembly'
+        obj_type = obj_info[2]
+        if 'KBaseMetagenomes.AnnotatedMetagenomeAssembly' not in obj_type:
+            raise ValueError(f"input ref '{ref}' is of type {obj_type}. function "
+                            "'get_annotated_metagenome_assembly' requires objects"
+                            " of type KBaseMetagenome.AnnotatedMetagenomeAssembly")
+
+    def get_annotated_metagenome_assembly(self, params):
+        """
+        params:
+            ref - workspace reference
+            included_fields - list of fields to include, defaults to list below if not specified.
+            included_feature_fields - list of fields to include in features
+
+        output
+        """
+        ref = params.get('ref', None)
+        included_fields = params.get('included_fields', None)
+        self._confirm_ws_type(ref)
+
+        included_feature_fields = params.get('included_feature_fields', [])
```

Comment:
is `included_feature_fields` used anywhere?

Reply:
Currently not, I added it in anticipation of returning some of the features. I'll get rid of since its not currently used